apps/modernization-ui/src/apps/page-builder/page/management/edit/content/PageContent.tsx

```diff
@@ -2,12 +2,16 @@ import { PagesTab } from 'apps/page-builder/generated';
 import { Sections } from '../section/Sections';
 import { PageSideMenu } from './PageSideMenu';
 import styles from './page-content.module.scss';
+import { useNavigate, useParams } from 'react-router-dom';
 
 type Props = {
     tab: PagesTab;
 };
 export const PageContent = ({ tab }: Props) => {
+    const navigateTo = useNavigate();
+    const { pageId } = useParams();
     const handleAddQuestion = (subsection: number) => {
+        navigateTo(`/page-builder/pages/${pageId}/question-library/${subsection}`);
         console.log('add question not yet implemented', subsection);
     };
```

Comment:
Question: Isn't the add question flow supposed to display within a modal?

Reply:
I have removed the model and displaying with in the page itself.

Reply:
Fixed to open in the model